### ISSUE TITLE
pygbag

### ISSUE THREAD

**Issue by pmp-p** [pygame-web/pkg-porting-wasm#3]
https://pypi.org/project/pygbag

https://github.com/pygame-web/pygbag

--

pygbag should be able to package from web for people that cannot install module or python , or have too old python on their workstation or mobile/smart devices.

**Comment by andreagalle** [comment]
I have been able to use pygbag in [this scenario](https://github.com/pygame-web/pygbag/issues/57) , in particular:
- within a Docker container
- within a GitHub codespace

I will open ASAP couple of PR to include these changes in the main repo. OK?

**Comment by pmp-p** [comment]
> I will open ASAP couple of PR to include these changes in the main repo. OK?

Sure anything that can help people packaging for web when they are stuck on device where capabalities have been limited is very welcome.

**Comment by andreagalle** [comment]
Concerning running it on GitHub Codespaces, it was running like a charm before Summer, then something broke: once it tries to GET the `archives/0.8/pythons.js` resource, I get the folllowing error

```
has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

The question is: how can I set this header for this particular resource, since it is actually set for other resources as well as `archives/0.8/pythonrc.py` or `archives/0.8//browserfs.min.js`

**Comment by pmp-p** [comment]
Try 0.8.1 for your use case. and if it works fine we will make a conditionnal + a cmd line option beetween 0.8.1 and 0.8.2 for 0.8.3 to cover all cases.

PS: maybe you could document basic GitHub Codespaces usage on [wiki](https://github.com/pygame-web/pygame-web.github.io) ?

**Comment by andreagalle** [comment]
I tried it out and unfortunately it is not working: thus, I already opened another PR https://github.com/pygame-web/pygbag/pull/119 with a couple questions.

Of course I will document on Codespaces usage on wiki, please could you even renew your past invitation (here below)? Unfortunately it expired, but I would like to join the https://github.com/pygame-web organization, but I wanted to!

![image](https://github.com/pygame-web/pkg-porting-wasm/assets/37024974/a5eb3907-f878-4ecc-a105-1927fbe0bb26)
